index.js

```diff
@@ -3,13 +3,22 @@
 const differenceWith = require('lodash/differenceWith');
 const isMatch = require('lodash/isMatch');
 const isPlainObject = require('lodash/isPlainObject');
+const isEqualWith = require('lodash/isEqualWith');
 
-function actionsMatch(actions, storeActions) {
+function actionsContaining(actions, storeActions) {
     return differenceWith(actions, storeActions, (action, storeAction) => isMatch(storeAction, action)).length === 0;
 }
 
-function waitForActions(store, actions, timeout) {
-    timeout = timeout || 50;
+function actionsMatchOrder(actions, storeActions) {
+    return isEqualWith(actions, storeActions, (action, storeAction) => isMatch(storeAction, action));
+}
+
+function waitForActions(store, actions, options) {
```

Comment:
actions -> expectedActions.

Reply:
Note that we don't need to change the README, I think `expectedActions` describes our intention better for someone that is trying to understand the code.